_includes/footer.html

```diff
@@ -0,0 +1,88 @@
+          <li><a href="http://jobs.azavea.com/">Jobs</a></li>
+          <li><a href="https://www.azavea.com/blog/">Blog</a></li>
+          <li><a href="https://www.azavea.com/press/">Press</a></li>
+        </ul>
+      </div>
+      <div class="column">
+        <h5 class="footer-header">Services</h5>
+        <ul class="footer-menu">
+          <li><a href="https://www.azavea.com/services/software-development/">Software Development</a></li>
+          <li><a href="https://www.azavea.com/services/data-analytics/">Data Analytics</a></li>
+          <li><a href="https://www.azavea.com/services/ux-design/">UX Design</a></li>
+          <li><a href="https://www.azavea.com/services/big-geospatial-data/">Big Geospatial Data</a></li>
+          <li><a href="https://www.azavea.com/services/research-and-development/">R&D</a></li>
+        </ul>
+      </div>
+      <div class="column">
+        <h5 class="footer-header">Newsletter</h5>
+        <div id="mc_embed_signup">
+          <form action="https://azavea.us1.list-manage.com/subscribe/post?u=61da999c9897859f1c1fff262&amp;id=d189bde191" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" class="validate" target="_blank" novalidate>
+            <div id="mc_embed_signup_scroll">
+              <div class="mc-field-group">
+                <label for="mce-EMAIL" class="sr-only">Email Address</label>
+                <input type="email" value="" name="EMAIL" class="required email" id="mce-EMAIL" placeholder="Email address">
+              </div>
+              <div id="mce-responses" class="clear">
+                <div class="response" id="mce-error-response" style="display:none"></div>
+                <div class="response" id="mce-success-response" style="display:none"></div>
+              </div>
+              <div style="position: absolute; left: -5000px;" aria-hidden="true"><input type="text" name="b_61da999c9897859f1c1fff262_d189bde191" tabindex="-1" value=""></div>
+              <input class="btn btn-primary" type="submit" value="Subscribe" name="subscribe" id="mc-embedded-subscribe">
+            </div>
+          </form>
+        </div>
+      </div>
+    </div>
+    <div class="row">
+      <div class="column footer-social">
+        <a href="http://twitter.com/azavea"><img src="{{ site.baseurl }}/assets/icons/icon-twitter.svg" alt="Azavea twitter"></img></a>
+        <a href="http://facebook.com/azavea"><img src="{{ site.baseurl }}/assets/icons/icon-facebook.svg" alt="Azavea facebook"></img></a>
+        <a href="http://www.linkedin.com/company/azavea"><img src="{{ site.baseurl }}/assets/icons/icon-linkedin.svg" alt="Azavea linkedin"></img></a>
+        <a href="http://github.com/azavea"><img src="{{ site.baseurl }}/assets/icons/icon-github.svg" alt="Azavea github"></img></a>
+      </div>
+    </div>
+    <div class="row">
+      <div class="column footer-bottom">
+        <a href="http://azavea.com/terms-of-use/">Terms of Use</a> •
```

Comment:
Nitpick, but we can make these HTTPS to avoid a redirect for whoever clicks.

Reply:
Will do